ios/MLCChat/app-config.json

```diff
@@ -1,27 +1,18 @@
 {
-  "model_libs": [
-    "Mistral-7B-Instruct-v0.2-q3f16_1",
-    "Llama-2-7b-chat-hf-q3f16_1",
-    "RedPajama-INCITE-Chat-3B-v1-q4f16_1"
-  ],
   "model_list": [
     {
-      "model_url": "https://huggingface.co/mlc-ai/mlc-chat-Llama-2-7b-chat-hf-q3f16_1/",
-      "local_id": "Llama-2-7b-chat-hf-q3f16_1"
+      "model_path": "Mistral-7B-Instruct-v0.2-q3f16_1",
+      "model_id": "Mistral-7B-Instruct-v0.2-q3f16_1",
+      "model_lib": "mistral_q3f16_1",
+      "model_lib_path": "lib/Mistral-7B-Instruct-v0.2/Mistral-7B-Instruct-v0.2-q3f16_1-iphone.tar",
+      "estimated_vram_bytes": 4000000000
```

Comment:
I wonder if we could have a more specific number using https://github.com/mlc-ai/mlc-llm/pull/1525. The estimate is calculated based on the metadata, so `-cuda.so` would give the same estimation.

Reply:
That makes sense, I will run that command to better report estimated mem usage. Thanks!